docs/src/parameter_distributions.md

```diff
@@ -361,28 +357,108 @@ end
 gif(anim_bounded, "anim_bounded.gif", fps = 10) # hide
 ```
 
-The following generates the transformed `Normal(0.5, 1)` distribution
 
+
+## Full Examples
+
+### Simple example - 1
```

Comment:
Once again, I think rearranging to put the recommended constructor first would be good here

Reply:
Addressed in [9b09cea](https://github.com/CliMA/EnsembleKalmanProcesses.jl/pull/161/commits/9b09cea7032cefb92cd0332b9656b924152356c1)